### ISSUE TITLE
performance worse than Mask R-CNN

### ISSUE THREAD

**Issue by eLeschke** [zhanggang001/RefineMask#27]
Hello,

First of all thank you for sharing your work!
I tried to train a model on my own dataset (wildlife animal data) with different hyperparameters but the performance does not reach Mask R-CNN. (0.66 AP with RefineMask against 0.7 AP with Mask R CNN)
I trained it on a single gpu.
In another issue you wrote that this isnt possible, could the lower batchsize be the reason for the bad performance?

Thank you in advance!

**Comment by zhanggang001** [comment]
> Hello,
> 
> First of all thank you for sharing your work! I tried to train a model on my own dataset (wildlife animal data) with different hyperparameters but the performance does not reach Mask R-CNN. (0.66 AP with RefineMask against 0.7 AP with Mask R CNN) I trained it on a single gpu. In another issue you wrote that this isnt possible, could the lower batchsize be the reason for the bad performance?
> 
> Thank you in advance!

1. I have never tried single GPU training on both Mask R-CNN and our proposed RefineMask, except for debugging or testing;

2. The small batch size may be the reason, you can try a larger batch size per GPU;

3. Quality of the mask annotations may also affect the Improvements of RefineMask, but it should not be worse than Mask R-CNN.
